### PR TITLE
embed you tube video on avalanche puzzle landing page

### PR DESCRIPTION
**Pull request by IvySwing**

### CODE REVIEW

**Review by chair28980 — CHANGES_REQUESTED** [review]
Great start! A couple notes.

looks like mobile view is broken


Screen Shot 2022-06-28 at 5 42 34 PM

- video is not quite centered on desktop
video-center

**Comment by IvySwing** [comment]
once review is completed and branch is merged, I will redeploy

File: pages/puzzle/[landing].tsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ import Wrapper from "@components/wrapper";
 import NavAvalanche from "@components/nav-avalanche";
 import Puzzle from "@components/puzzle";
 import Markdown from "@components/markdown";
+import Video from "public/puzzles/avalanche/video";
```
Comment (stormcloud266) [inline pages/puzzle/[landing].tsx]: Everything is looking super good! The only thing, I don't think we need this import here anymore.

**Review by stormcloud266 — CHANGES_REQUESTED** [review]

File: pages/puzzle/[landing].tsx (hunk 1 above)
Comment (chair28980) [reply pages/puzzle/[landing].tsx]: @IvySwing please get this change in and re-submit for review, should be g2g from there.

**Comment by IvySwing** [comment]
@chair28980 @stormcloud266 I made the changes and re-requested reviews. Let me know when I can merge

**Review by chair28980 — APPROVED** [review]
Awesome stuff! LGTM! 🍏

**Comment by stormcloud266** [comment]
@IvySwing This is looking awesome!

**Review by stormcloud266 — APPROVED** [review]

**Pull request merged**